docs/further_reading.bib

```diff
@@ -278,3 +300,57 @@ @article{HIRTH2020100433
             accessible free of cost, it must also come with an open data license, the
             most common types of which are also reviewed in this paper.}
 }
+
+@book{10.1145/3310205,
+author = {Ilyas, Ihab F. and Chu, Xu},
+title = {Data Cleaning},
+year = {2019},
+isbn = {9781450371520},
+publisher = {Association for Computing Machinery},
+address = {New York, NY, USA},
+doi = {10.1145/3310205,},
+url = {https://dl.acm.org/doi/book/10.1145/3310205},
+abstract = {Data quality is one of the most important problems in data management, since
+    dirty data often leads to inaccurate data analytics results and incorrect business
+    decisions. Poor data across businesses and the U.S. government are reported to cost
+    trillions of dollars a year. Multiple surveys show that dirty data is the most
+    common barrier faced by data scientists. Not surprisingly, developing effective and
+    efficient data cleaning solutions is challenging and is rife with deep theoretical
+    and engineering problems.This book is about data cleaning, which is used to refer to
+    all kinds of tasks and activities to detect and repair errors in the data. Rather
+    than focus on a particular data cleaning task, we give an overview of the endto- end
+    data cleaning process, describing various error detection and repair methods, and
+    attempt to anchor these proposals with multiple taxonomies and views. Specifically,
+    we cover four of the most common and important data cleaning tasks, namely, outlier
+    detection, data transformation, error repair (including imputing missing values),
+    and data deduplication. Furthermore, due to the increasing popularity and
+    applicability of machine learning techniques, we include a chapter that specifically
+    explores how machine learning techniques are used for data cleaning, and how data
+    cleaning is used to improve machine learning models.This book is intended to serve
+    as a useful reference for researchers and practitioners who are interested in the
+    area of data quality and data cleaning. It can also be used as a textbook for a
+    graduate course. Although we aim at covering state-of-the-art algorithms and
+    techniques, we recognize that data cleaning is still an active field of research and
+    therefore provide future directions of research whenever appropriate.}
+}
+
+@book{doi:https://doi.org/10.1002/0470036427,
+publisher = {John Wiley & Sons, Ltd},
+isbn = {9780470036426},
+author = {von Meier, Alexandra},
+title = {Electric Power Systems: A Conceptual Introduction},
+doi = {https://doi.org/10.1002/0470036427},
```

Comment:
Should this just be `10.1002/0470036427` like the other ones?

Reply:
Bah yes.